### PR TITLE
media-sound/ncmpcpp: Include upstream patch to genius lyrics

### PR DESCRIPTION
**Pull request by spencermp**
The master branch has had this fix to lyric fetching from Genius for about 2 years: github.com/ncmpcpp/ncmpcpp/commit/df1c4d7

The patch works, but no new version has been pushed since that commit, so this is just a backport of that fix.

### CODE REVIEW

**Comment by NexAdn** [comment]
Hm, I'd rather have a _p release pointing to the most recent commit on master published. This way, we can include all the other “recent” patches as well.

**Comment by spencermp** [comment]
> Hm, I'd rather have a _p release pointing to the most recent commit on master published. This way, we can include all the other “recent” patches as well.

There is the git version 9999 ebuild but I always am hesitant to unblock that sort of thing. Not 100% logical, but I find it much better to just use amd64 or ~amd64 packages without the more "experimental" feel of vcs packages.

At the same time, the updates for ncmpcpp seemed to have stopped around a year ago, Dec 31, 2022

File: media-sound/ncmpcpp/ncmpcpp-0.9.2-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+EAPI=7
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="https://ncmpcpp.rybczak.net/ https://github.com/ncmpcpp/ncmpcpp"
+SRC_URI="https://rybczak.net/ncmpcpp/stable/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```
Comment (mm1ke) [inline media-sound/ncmpcpp/ncmpcpp-0.9.2-r4.ebuild:12]: should be `GPL-2+`, see: https://github.com/ncmpcpp/ncmpcpp/blob/master/src/ncmpcpp.cpp#L5-L8

**Comment by NexAdn** [comment]
> There is the git version 9999 ebuild but I always am hesitant to unblock that sort of thing. Not 100% logical, but I find it much better to just use amd64 or ~amd64 packages without the more "experimental" feel of vcs packages.

Hm, maybe I chose my words poorly. I meant a _p release pointing to the most recent commit at the time the ebuild is written, not a live ebuild that fetches whatever the most recent commit is at the time the ebuild is merged. That's not so “live”y, but just creating something like a release independently from upstream.

I'll write something up an open a new PR.

**Comment by a17r** [comment]
@NexAdn: A snapshot should always be last resort; if we can not expect upstream to cut a new release in time for our needs (ideally after having inquired about it) or they are unresponsive.

**Comment by spencermp** [comment]
I'm going to close this one for now, in the meantime I'll wait to see what happens with the point release.

**Comment by NexAdn** [comment]
> @NexAdn: A snapshot should always be last resort; if we can not expect upstream to cut a new release in time for our needs (ideally after having inquired about it) or they are unresponsive.

@a17r there are already open issues requesting new releases and upstream is unresponsive, see also https://github.com/ncmpcpp/ncmpcpp/issues/576

**Pull request closed without merging**